d/Dockerfile

```diff
@@ -10,12 +10,13 @@ RUN apt-get -qy install build-essential ldc dub openssl libssl-dev zlib1g-dev ll
 {{/files}}
 
 ENV DC=ldc2
-RUN dub build -b release --compiler=ldc2
+RUN dub build -b release-nobounds --compiler=ldc2
 
 FROM debian:stable
 
 WORKDIR /opt/web
 
+RUN echo "* hard nofile 16768" >> /etc/limits.conf
```

Comment:
Do you mean you want to override distribution limits ?

What is the purpose?

Reply:
Many distros have a very low settings for max number of open descriptors (files, sockets, and so on) and since we're testing a lot of concurrent connections this could lead to errors.

Reply:
That's fine then. Usually it is really lower (1024 or 512)